<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">lineEdit{
height:50
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 94, 94);
}</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="todo_listWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(253, 255, 203);
}
QListWidget{
background-color:#607EAA;
margin:10%;
margin-right:10%;rgb(255, 255, 190)
font: 12pt &quot;Comic Sans MS&quot;;
}
QPushButton{
background-color:#607EAA;
margin-left:10%;
margin-right:10%;
font:14pt;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="unselect_button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
}</string>
        </property>
        <property name="text">
         <string>Unselect all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
}</string>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finish_button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(62, 215, 72)
}</string>
        </property>
        <property name="text">
         <string>Finish </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
